testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksCleanUpGlueMetastore.java

```diff
@@ -1,83 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.tests.product.deltalake;
-
-import com.amazonaws.services.glue.AWSGlueAsync;
-import com.amazonaws.services.glue.AWSGlueAsyncClientBuilder;
-import com.amazonaws.services.glue.model.Database;
-import com.amazonaws.services.glue.model.DeleteDatabaseRequest;
-import com.amazonaws.services.glue.model.EntityNotFoundException;
-import com.amazonaws.services.glue.model.GetDatabasesRequest;
-import com.amazonaws.services.glue.model.GetDatabasesResult;
-import io.airlift.log.Logger;
-import io.trino.plugin.hive.metastore.glue.AwsApiCallStats;
-import io.trino.tempto.ProductTest;
-import org.testng.annotations.Test;
-
-import java.util.List;
-
-import static com.google.common.collect.ImmutableList.toImmutableList;
-import static io.trino.plugin.hive.metastore.glue.AwsSdkUtil.getPaginatedResults;
-import static io.trino.tests.product.TestGroups.DELTA_LAKE_DATABRICKS;
-import static io.trino.tests.product.TestGroups.PROFILE_SPECIFIC_TESTS;
-import static java.lang.System.currentTimeMillis;
-import static java.util.concurrent.TimeUnit.DAYS;
-
-public class TestDeltaLakeDatabricksCleanUpGlueMetastore
-        extends ProductTest
-{
-    private static final Logger log = Logger.get(TestDeltaLakeDatabricksCleanUpGlueMetastore.class);
-    private static final String TEST_DATABASE_NAME_PREFIX = "test_";
-
-    @Test(groups = {DELTA_LAKE_DATABRICKS, PROFILE_SPECIFIC_TESTS})
-    public void testCleanupOrphanedDatabases()
```

Comment:
> There exists already a test doing the same thing in the class

> `io.trino.plugin.deltalake.metastore.glue.TestDeltaLakeCleanUpGlueMetastore`


product tests and integration tests are run independently
it may be that one is run while the other is not

i think this is why we had the cleanup in two places.

Reply:
it may also be that the product tests run on a different AWS account than the integration tests.